tests/tck/features/match/With.feature

```diff
@@ -387,3 +387,18 @@ Feature: With clause
       | [:teammate "Tim Duncan"->"Tony Parker" @0 {end_year: 2016, start_year: 2001}]       |
       | [:like "Tim Duncan"->"Tony Parker" @0 {likeness: 95}]                               |
       | [:teammate "Tim Duncan"->"Danny Green" @0 {end_year: 2016, start_year: 2010}]       |
+      | [:teammate "Tim Duncan"->"Manu Ginobili" @0 {end_year: 2016, start_year: 2002}]     |
+      | [:like "Tim Duncan"->"Manu Ginobili" @0 {likeness: 95}]                             |
+
+  Scenario: with wildcard after multiple matches
+    When executing query:
+      """
+      match (v0:player)--(v1:team) where v1.team.name == "Spurs" and v0.player.name == "Tim Duncan"
+      match (v:player) where v.player.name != "Tim Duncan" with v0 where v0.player.age > 0
+      match (v0:player)
+      with *
+      return count(v0)
```

Comment:
could you test the following scenario case? 

```
match (v:player)
with v AS p
match (p)
with p AS v
match (v)
with *
return count (p)
```

Reply:
SemanticError: Alias used but not defined: `p'

Reply:
neo4j reports `Variable `p` not defined`

Reply:
Since p has been aliased as v in `with p AS v`, I think it is good here.

Reply:
Let me add this as a tck though.